genstudio-io-runtime-app/README.md

```diff
@@ -0,0 +1,83 @@
+# GenStudio MLR Claims App
+
```

Comment:
Might worth including links to io action logs or related.
https://developer.adobe.com/app-builder/docs/guides/application_logging/